frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts

```diff
@@ -0,0 +1,198 @@
+import { safeLoadAll } from 'js-yaml';
+import { checkErrors } from '../../support';
+import { isLocalDevEnvironment } from '../../views/common';
+import { detailsPage } from '../../views/details-page';
+import { infoMessage, hintMessage, warningMessage, warningToast } from '../../views/form';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+import { nav } from '../../views/nav';
+
+const PLUGIN_NAME = 'console-demo-plugin';
+const PLUGIN_PATH = '../../../dynamic-demo-plugin';
+const CHECK_UPDATE_WAIT = 30000;
+const PLUGIN_PULL_SPEC = Cypress.env('PLUGIN_PULL_SPEC');
```

Comment:
I'm hesitant adding `PLUGIN_PULL_SPEC` as cypress env since it is really specific to plugin cases while for variables becoming cypress env they should have global effect

Reply:
@yapei yes we need this env variable because there will be another pr in the release repository. That pr is going to set the image pull spec in the environment variable for this test. The environment variable will be set during the builds for e2e testing in that release repository. Thats why we are grabbing that pull spec here. You will see that pr coming soon and see how the pull spec is passed through the env var.